src/utils/explorer-api.js

```diff
@@ -54,3 +54,32 @@ export async function getTransactions(accountId) {
 }
 
 export const transactionExtraInfo = (hash, signer_id) => wallet.connection.provider.sendJsonRpc('tx', [hash, signer_id])
+
+export async function getAccountId(publicKey) {
+    if (!publicKey) return {}
+
+    const accountId = await new Promise((resolve, reject) => wamp.call(
+        `${WAMP_NEAR_EXPLORER_TOPIC_PREFIX}.select`,
+        [
+            `
+                SELECT
+                    account_id
+                FROM 
+                    access_keys
+                WHERE 
+                    public_key = :publicKey
+            `,
+            { publicKey }
+        ],
+        {
+            onSuccess(dataArr) {
```

Comment:
nit: looks like it makes sense to make a wrapper function for making `wamp.call` with all the boilerplate related to `Promise` and `onSuccess`, etc. E.g. I used this in other project:

```js
    const queryExplorer = (sql, params) => new Promise((resolve, reject) => wamp.call(
        `${WAMP_NEAR_EXPLORER_TOPIC_PREFIX}.select`,
        [ sql, params ],
        {
            onSuccess(dataArr) {
                resolve(dataArr[0]);
            },
            onError(err) {
                reject(err);
            }
        }
    ));
```